studio/settings.py

```diff
@@ -403,62 +408,62 @@
             "()": structlog.stdlib.ProcessorFormatter,
             "processor": structlog.processors.JSONRenderer(),
         },
-        "plain_console": {
-            "()": structlog.stdlib.ProcessorFormatter,
-            "processor": structlog.dev.ConsoleRenderer(),
-        },
-        "key_value": {
-            "()": structlog.stdlib.ProcessorFormatter,
-            "processor": structlog.processors.KeyValueRenderer(key_order=["timestamp", "level", "event", "logger"]),
-        },
         "colored": {
-            "()": colorlog.ColoredFormatter,  # colored output
+            "()": colorlog.ColoredFormatter,
             "datefmt": "%Y-%m-%d %H:%M:%S",
-            "format": "%(log_color)s%(asctime)s - %(levelname)s - %(module)s: %(message)s%(reset)s",
+            "log_colors": {
+                "DEBUG": "blue",
+                "INFO": "green",
+                "WARNING": "yellow",
+                "ERROR": "red",
+                "CRITICAL": "bold_red",
+            },
+            "format": "%(log_color)s%(asctime)s - %(levelname)s - %(name)s: %(message)s%(reset)s",
         },
     },
     "handlers": {
         "console": {
             "class": "logging.StreamHandler",
             "formatter": "colored",
         },
-        "json_file": {
-            "class": "logging.handlers.WatchedFileHandler",
-            "filename": "logs/json.log",
+        "json": {
+            "class": "logging.StreamHandler",
             "formatter": "json_formatter",
         },
-        "flat_line_file": {
-            "class": "logging.handlers.WatchedFileHandler",
-            "filename": "logs/flat_line.log",
-            "formatter": "key_value",
-        },
     },
     "loggers": {
-        "django_structlog": {
-            "handlers": ["json_file"],
-            "level": "DEBUG" if DEBUG else "WARNING",
-        },
-        # Make sure to replace the following logger's name for yours
-        "portal": {
-            "handlers": ["console"],
-            "level": "DEBUG" if DEBUG else "WARNING",
+        "": {
+            "handlers": ["console" if DEBUG else "json"],
+            "level": "DEBUG" if DEBUG else "INFO",
         },
+        'django.server': {
+            'handlers': ['console'],
+            'level': "DEBUG" if DEBUG else "INFO",
```

Comment:
I would a couple of lines about this in README, that "Use INFO level by default, but if you want to add some debug messages, use DEBUG. But keep in mind, that logs should be equally useful locally and in production, so don't overuse DEBUG level"

Reply:
Or here might actually be a proper place for this
https://scilifelab.atlassian.net/wiki/x/DIBNlg

But for convenience, we could write a few words here as well.